sos/report/plugins/openstack_instack.py

```diff
@@ -57,6 +59,20 @@ def setup(self):
         vars_any = [p in os.environ for p in [
                     'OS_TENANT_NAME', 'OS_PROJECT_NAME']]
 
+        # capture yaml files to define overrides
+        uc_config = configparser.ConfigParser()
+        try:
+            uc_config.read(UNDERCLOUD_CONF_PATH)
+            override_opts = ['hieradata_override', 'net_config_override']
+            for opt in override_opts:
+                p = override_opts.get(opt)
+                if p:
+                    if not os.path.isabs(p):
+                        p = os.path.join('/home/stack', p)
+                    self.add_copy_spec([p])
```

Comment:
`add_copy_spec()` takes a single item as well. No need to cast this to a list here.